buildpacks/maven/tests/settings_xml_tests.rs

```diff
@@ -0,0 +1,212 @@
+use indoc::formatdoc;
+use libcnb_test::{
+    assert_contains, assert_not_contains, BuildpackReference, PackResult, TestConfig, TestRunner,
+};
+use std::fs;
+use std::path::PathBuf;
+
+#[test]
+fn maven_settings_url_success() {
+    TestRunner::default().run_test(
+        default_config().env("MAVEN_SETTINGS_URL", SETTINGS_XML_URL),
+        |context| {
+            assert_contains!(
+                context.pack_stdout,
+                &format!(
+                    "[BUILDPACK INTEGRATION TEST - SETTINGS TEST VALUE] {SETTINGS_XML_URL_VALUE}"
+                )
+            );
+        },
+    );
+}
+
+#[test]
+fn maven_settings_url_failure() {
+    TestRunner::default().run_test(
+            default_config()
+                .env("MAVEN_SETTINGS_URL", SETTINGS_XML_URL_404)
+                .expected_pack_result(PackResult::Failure),
+            |context| {
+                assert_contains!(
+                    context.pack_stderr,
+                    &format!("You have set MAVEN_SETTINGS_URL to \"{SETTINGS_XML_URL_404}\". We tried to download the file at this\nURL, but the download failed. Please verify that the given URL is correct and try again.")
```

Comment:
(Optional) Since this is a multi-line string, using `indoc` seems like it would be easier to read? Indoc provides `formatdoc!` which allows for formatting too:
https://docs.rs/indoc/latest/indoc/#formatting-macros